src/App.js

```diff
@@ -1,12 +1,177 @@
 import React from 'react';
+import {useForm} from 'react-hook-form'
 import './App.css';
 
+import FruitCounter from './components/FruitCounter';
+import LabelInput from "./components/LabelInput";
+
+import strawberry from './assets/strawberry.png'
+import banana from './assets/banana.png'
+import appel from './assets/appel.jpg'
+import kiwi from './assets/kiwi.png'
+
 function App() {
+    const {register, handleSubmit} = useForm();
+
+    function onFormSubmit(data) {
+        console.log(data);
+        console.log({Aardbeien: strawberryCounter, Bananen: bananaCounter, Appels: appelCounter, Kiwis: kiwiCounter});
+    }
+
+    const [strawberryCounter, setStrawberryCounter] = React.useState(0);
+    const [bananaCounter, setBananaCounter] = React.useState(0);
+    const [appelCounter, setAppelCounter] = React.useState(0);
+    const [kiwiCounter, setKiwiCounter] = React.useState(0);
+
+   function resetButton() {
+       setStrawberryCounter(0);
+       setBananaCounter(0);
+       setAppelCounter(0);
+       setKiwiCounter(0);
+       }
+
   return (
     <>
       <h1>Fruitmand bezorgservice</h1>
+
+        <div className="fruit">
```

Comment:
Ik zie dat deze div met classname een aantal keer herhaalt wordt. Had je deze dan niet beter in het FruitCounter component kunnen plaatsen?